liqd_product/assets/scss/style.scss

```diff
@@ -58,3 +57,6 @@
 
 @import '~a4-meinberlin/meinberlin/assets/scss/utility';
 @import '~a4-meinberlin/meinberlin/assets/scss/shame';
+
+@import "layout";
+@import "components/hero-unit";
```

Comment:
Since we long-term want to replace all those imports, at least the ones necessary, I said it'd be okay to just put them at the end.